fidesctl/src/fidesctl/core/config.py

```diff
@@ -104,3 +103,13 @@ def get_config(config_path: str = "") -> FidesctlConfig:
                 echo_red(f"Error reading config file from {file_location}")
     fidesctl_config = FidesctlConfig()
     return fidesctl_config
+
+def is_test_mode_enabled() -> bool:
```

Comment:
let's move this to a validation step within the CLI settings model. The config can automatically set the database_url to the right string at run time so nothing downstream actually has to do any extra checks, and you can pull the config from anywhere and get the right values. check the other pydantic models in `fideslang` for examples of doing validation and setting values accordingly. You'll definitely get into some pydantic weeds here but I think its a more elegant solution

Reply:
That's a good idea, I was thinking of doing something like this but was concerned about whether it could be misleading to reference database_url from the config object and potentially get test_database_url. 

I'm happy to make this change though, but in the current main.py I only create a database if it is in test mode. Do you think we even need this check?

Reply:
yeah we can change that to just check the env var directly I think, no need for the extra function

Reply:
oh wait, your point is that the check is kinda useless if we're doing the `database_exists` check anyway....

yeah in that case we can just drop that check altogether, and it'll do a check/create for the database url every time. I think thats a pretty solid solution and makes the users' lives easier as well

Reply:
I was actually thinking that calling `create-db` could be reserved for testing only. You're right that we do check if it exists but I feel that for standard use cases we shouldn't be creating the database for users. Not a strong opinion on this though.